docs/en/advanced/Namespace-Management.md

```diff
@@ -288,7 +288,10 @@ hello
 
 ### Metadata Active Sync for HDFS
 In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
-The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master. 
+Because active sync feature depends on HDFS events, this feature is only available when the UFS HDFS versions is later than 2.6.1.
+Be sure to specify a value for `alluxio.underfs.version` in your configuration file. It defaults to 2.2, which does not support HDFS events.
+Please refer to [HDFS Under Store]({{ '/en/ufs/HDFS.html?#supported-hdfs-versions' | relativize_url }}) for a list of supported Hdfs versions.
```

Comment:
still have a hanging `?`. It shouldn't affect the user's experience, but probably better to clean up the link.

Reply:
fixed